cupyx/runtime.py

```diff
@@ -134,6 +144,14 @@ def __init__(self):
                 nccl_runtime_version = '(unknown)'
             self.nccl_runtime_version = nccl_runtime_version
 
+        if cub is not None:
```

Comment:
How about using `cupy.cuda.cub_enabled` simply?

Reply:
It was to maintain consistency with the above checks 😅

Reply:
Yeah I think here we want to know if the module is built, not if it is enabled.

Reply:
I didn‘t thought about it that way but makes complete sense